src/publish.proj

```diff
@@ -24,4 +28,20 @@
     <!-- skip uploadToAzure when there are no nupkgs -->
     <CallTarget Targets="UploadToAzure" Condition="'@(_PackagesToPublish)' != ''" />
   </Target>  
-</Project>
+  
+  <Target Name="PublishToAzureBlobFeed" >
+    <PropertyGroup>
+      <PackageDownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)AzureTransfer\$(ConfigurationGroup)</PackageDownloadDirectory>
+      <FinalPublishPattern>$(PackageDownloadDirectory)\**\*.nupkg</FinalPublishPattern>
+      <FinalSymbolsPackagesPattern>$(PackageDownloadDirectory)\**\*.symbols.nupkg</FinalSymbolsPackagesPattern>
```

Comment:
When going to AzureBlobFeed shouldn't we be pushing the symbol packages somewhere as well?

Reply:
Add an explicit step below. 
Currently though these packages will be pushed to a normal nuget package feed, once we move to using the sleet implementation I think the actual symbol package feed will be functional. /cc @dagood

Reply:
I can maybe move the PropertyGroup for each out. To note we need to pass in an explicit symbol feed through PB_AzureSymbolFeedUrl